hapi-fees/src/main/java/com/hedera/services/usage/crypto/CryptoTransferUsage.java

```diff
@@ -65,7 +65,7 @@ public FeeData get() {
 		xferBytes += (hbarXfers + tokenXfers) * usageProperties.accountAmountBytes();
 		usageEstimator.addBpt(xferBytes);
 		if (hbarXfers > 0) {
-			addRecordRb(hbarXfers * usageProperties.accountAmountBytes());
+			addRecordRb((long) hbarXfers * usageProperties.accountAmountBytes());
```

Comment:
Doesn't require a cast to long as it's already a long.

Reply:
Fixed . Thanks !